src/shared/elf-util.c

```diff
@@ -788,11 +789,15 @@ int parse_elf_object(int fd, const char *executable, bool fork_disable_dump, cha
          * bound since the core files have an upper size limit. It's also not doing any
          * system call or interacting with the system in any way, besides reading from
          * the file descriptor and writing into these four pipes. */
+        int fork_flags = FORK_RESET_SIGNALS|FORK_CLOSE_ALL_FDS|FORK_WAIT|FORK_REOPEN_LOG;
+        if (userns_supported()) {
+                fork_flags |= FORK_NEW_USERNS|FORK_NEW_MOUNTNS|FORK_MOUNTNS_SLAVE;
+        }
```

Comment:
Redundant `{}`.

Not a matter, but we usually write something like the following:
```
ForkFlags flags = FORK_RESET_SIGNALS|FORK_CLOSE_ALL_FDS|FORK_WAIT|FORK_REOPEN_LOG|
        (userns_supported() ? FORK_NEW_USERNS|FORK_NEW_MOUNTNS|FORK_MOUNTNS_SLAVE : 0);
```

Reply:
The reason I did not use a ternary operator is the improved readability so I would prefer to leave it as-is. I removed the `{}`.